### PR TITLE
Readme: Add a related media section and tidy up the news a bit. 

### PR DESCRIPTION
**Pull request by PhilipMetzger**
This bundles all Jujutsu related media (the talk, its slides and the three articles we were featured in) 
into a section.

Also update the news section to mention the recent deprecations and the three articles we recently 
were featured in. 

Thanks to everyone who wrote one.

[Rendered](https://github.com/martinvonz/jj/tree/push-mtrqnqsyrwrt)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +130,12 @@ developers monitor both channels.
 
 ### News and Updates 📣
 
+- **Feb 2024**: Chris Krycho published an article about Jujutus called [jj init][krycho]
+  and Steve Klabnik followed up with the [Jujutsu Tutorial][klabnik]. 
+  And Version 0.14 is released, which deprecates [jj checkout and merge](changelog.md#deprecations),
+  and `jj init --git`.
```
[the remark this answers is not in the document]
Comment (ilyagr) [reply README.md:136]: I'm not sure, but I'm wondering whether it'd be better to have a separate Bibliography/"Articles about Jujutsu" section.

[the remark this answers is not in the document]
Comment (PhilipMetzger) [reply README.md:136]: I think the related media section we talked about on [Discord](https://discord.com/channels/968932220549103686/968932220549103689/1202361147907641364) should be nice.

[the remark this answers is not in the document]
Comment (PhilipMetzger) [reply README.md]: PTAL at the new section.

[the remark this answers is not in the document]
Comment (ilyagr) [reply README.md]: I would just move the changelog entries related to the articles into the new section. Essentially, the "news" section would be split into two. Both the date and the description seem helpful in the "Articles" (or "Related Media") section, and then there's no need to duplicate them in "news".

File: README.md [hunk 2]
```diff
@@ -130,13 +130,33 @@ developers monitor both channels.
 
 ### News and Updates 📣
 
+- **Feb 2024**: Version 0.14 is released, which deprecates [jj checkout and merge](changelog.md#deprecations),
```
Comment (martinvonz) [inline README.md:133]: nit: Style it as "`jj checkout` and `jj merge`" for clarity?

File: README.md [hunk 3]
```diff
@@ -130,13 +130,33 @@ developers monitor both channels.
...
+  and `jj init --git`, which is now just called `jj git init`.
```
Comment (martinvonz) [inline README.md:134]: nit: maybe ", as well as `jj init --git`, which ..." to try to make it clearer that the `jj init --git` bit is distinct from the `jj checkout/merge`

File: README.md [hunk 4]
```diff
@@ -130,13 +130,33 @@ developers monitor both channels.
 
 ### News and Updates 📣
 
+- **Feb 2024**: Version 0.14 is released, which deprecates [jj checkout and merge](changelog.md#deprecations),
+  and `jj init --git`, which is now just called `jj git init`. 
+- **Feb 2024**: Chris Krycho published an article about Jujutsu called [jj init][krycho]
+  and Steve Klabnik followed up with the [Jujutsu Tutorial][klabnik]. 
+- **Jan 2024**: Jujutsu was featured in an LWN.net article, called
```
Comment (martinvonz) [inline README.md:137]: nit: drop the comma? I find it easier to read without it, and you don't have a comma before "called" on the line above

File: README.md (hunk 2 above)
Comment (PhilipMetzger) [reply README.md]: done.

File: README.md (hunk 3 above)
Comment (PhilipMetzger) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (PhilipMetzger) [reply README.md]: done

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (PhilipMetzger) [reply README.md]: I've still duplicated them there but I want to hear @thoughtpolice opinion on it, as the news section was his idea.

File: README.md [hunk 5]
```diff
@@ -130,13 +130,35 @@ developers monitor both channels.
...
+  [Jujutsu: a new, Git-compatible version control system][lwn].
 - **Oct 2023**: Version 0.10.0 is released! Now includes a bundled merge and
   diff editor for all platforms, "immutable revsets" to avoid accidentally
   `edit`-ing the wrong revisions, and lots of polish.
 - **Jan 2023**: Martin gave a presentation about Google's plans for Jujutsu at
   Git Merge 2022! See the
-  [slides](https://docs.google.com/presentation/d/1F8j9_UOOSGUN9MvHxPZX_L4bQ9NMcYOp1isn17kTC_M/view)
-  or the [recording](https://www.youtube.com/watch?v=bx_LGilOuE4).
+  [slides][merge-slides]
+  or the [recording][merge-talk].
```
Comment (thoughtpolice) [inline README.md:145]: nit: go ahead and remove the superfluous newlines, please, e.g. `See the [slides][merge-slides] or the [recording][merge-talk].`

**Review by thoughtpolice — APPROVED** [review]
lgtm, just a minor nit

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (thoughtpolice) [reply README.md]: Yeah, I think splitting it up makes sense.

[the remark this answers is not in the document]
Comment (PhilipMetzger) [reply README.md]: Ok, I moved the articles from the news to the new section just like ilya suggested.

File: README.md (hunk 5 above)
Comment (PhilipMetzger) [reply README.md]: done

**Pull request merged**